docs/release-notes/software-releases.md

```diff
@@ -3,17 +3,34 @@ id: software-releases
 title: Software Releases
 ---
 
-A DCC software release is a set of new features and bug fixes for the ARGO Data Platform. Software releases happen approximately every month. 
+A DCC software release is a set of new features and bug fixes for the ARGO Data Platform. Software releases happen approximately every month.
 
-## Software Release 1.0 
+<!---
+## Software Release 2.0
 
 **Release Date:**
 
-### New Features 
-Coming soon.
+### Bug Fixes
 
-### Bug Fixes 
-Coming soon.
+None to report.
 
-### Known Issues 
-Coming soon.
\ No newline at end of file
+### Known Issues
+
+None to report.
+------>
+
+## Initial Software Release
+
+Data Platform v1.54.0 -
+API v3.2.0
+
+**Release Date:** June 19, 2020
+
+### New Features
+
+This is the first release of the ARGO Data Platform. Major features include:
+
+- **Clinical Data Submission**: The ARGO Data Platform is open for data submission for registered programs. Registered programs can begin [submitting clinical data](/docs/submission/submitting-clinical-data) through the Program Services area.
+- **Molecular Data Submission**: The ARGO Data Platform is open for data submission for registered programs. Registered programs can begin [submitting molecular data](/docs/submission/submitting-molecular-data) through the Song and Score clients.
+- **Program Management**: For submitting programs, access to program submission and data can be managed from the Program Services area. This includes the ability for program administrators to review a program profile, as well as add and remove program members.
+- **File Repository**: Users can browse the complete ARGO data set using the [File Repository](https://platform.icgc-argo.org/repository). Files of interest can be [downloaded](/docs/data-access/data-download), provided that access to controlled data has been confirmed. To access controlled data, please see the [DACO application process here](/docs/data-access/data-access), or [apply for DACO approval here](https://icgc.org/daco).
```

Comment:
Do you want to mention anything about the Program Dashboard in these points?

Reply:
no i don't think so - its more high level